README.md

```diff
@@ -1,83 +1,80 @@
-Scel
-====
+# scel - sclang-mode for emacs
 
 SuperCollider/Emacs interface
 
+## Installation
 
-Installation requirements
--------------------------
+The repository contains two subprojects. `/sc` contains the SuperCollider code
+required to implement the emacs interface. `/el` contains the emacs-lisp
+implementation of the mode. Emacs and SuperCollider each have their own package
+managers, so it is required to install each half separately.
 
-For the HTML help system, you will need emacs-w3m support.
+### Installing scel quark
 
+The `scel` Quark is required for emacs to communicate with sclang. Evaluate this in the SuperCollider GUI:
 
-Installation (default)
-----------------------
-
-By default emacs-lisp files are installed in
-
-`$prefix/share/emacs/site-lisp`
-
-SuperCollider files are put in
+``` supercollider
+Quarks.install("https://github.com/supercollider/scel");
+```
 
-`$prefix/share/SuperCollider/Extensions/scide_scel`
```

Comment:
Great! This works, BUT:
1. SuperCollider needs to be set to use the default `yaml` file, OR sc-el needs to be set to use the other/specified `yaml` file.
- SC IDE breaks after installing the quark (after sclang reboot) - we get duplicated classes

Re 1: IMO this needs to be spelled out in the README: when adding a quark, it's being added to the "compile directories" in the yaml file, the one that's specified in the SC IDE. The user either needs to use the default yaml file, or specify the desired yaml file on the sc-el side. It would be great to add a note somewhere (separate paragraph probably, referenced here maybe?) how to specify the yaml file to be used by sc-el

Re 2: When installing files with CMake, [they go into scide_scel folder](https://github.com/jxa/scel/blob/packaging/sc/CMakeLists.txt#L6). AFAIU the folder name is how IDEs recognize which classes to load/not load. In my testing it seems that renaming the folder `sc` to `scide_scel` in the scel repo fixes this. If that indeed is the solution, [this](https://github.com/jxa/scel/blob/packaging/CMakeLists.txt#L4) also needs to be updated accordingly and installing with cmake needs to be tested again.
BTW would it be too much to add installation with CMake to GHA? That's kind of involved since you'd need to install whole SC first), but maybe would be a good addition? That's rather optional, and could be easily added later. But fixing the issue with duplicated classes in SC-IDE definitely needs to be addressed.

Reply:
After thinking a little more about this, adding Cmake install (and SC install) to GHA should not be part of this PR - to be possibly considered later. On thing to consider would be to run such test on the main SC repo, where we already build SC.

Reply:
I see there is a variable `sclang-library-configuration-file` that can be set by the emacs user if they want to customize the yaml file to load. I can add some text to the readme to clarify this.

It took me a little while to understand what was happening in (2), but your suggestion appears to work. I'm close to shipping that change, I just need to verify that it still works when compiling from source. (which I need to do in a VM because I'm on a M1 macbook :-/ )

Reply:
I was able to confirm that after renaming the `sc` directory, installation from source works in both Emacs and SC IDE with no errors.